lib/openhab/dsl/monkey_patch/items/persistence.rb

```diff
@@ -46,12 +46,9 @@ def previous_state(service = nil, skip_equal: false)
           ].each do |method|
             define_method(method) do |timestamp, service = nil|
               service ||= persistence_service
-              if timestamp.is_a? Java::JavaTimeTemporal::TemporalAmount
-                timestamp = Java::JavaTime::ZonedDateTime.now.minus(timestamp)
-              end
-              result = PersistenceExtensions.public_send(method, self, timestamp, service&.to_s)
-              if result.is_a?(Java::OrgOpenhabCoreLibraryTypes::DecimalType) && respond_to?(:unit) && unit
-                Quantity.new(Java::OrgOpenhabCoreLibraryTypes::QuantityType.new(result.to_big_decimal, unit))
+              result = PersistenceExtensions.public_send(method, self, to_zdt(timestamp), service&.to_s)
+              if %w[average_since delta_since deviation_since sum_since variance_since].include? method
```

Comment:
Could you make this a constant instead?

Reply:
> Could you make this a constant instead?

Are you referring to %w? I've replaced them with %i.